### ISSUE TITLE
Undefined is not an object (tensorflow ImageRecognition)

### ISSUE THREAD

**Issue by SpQuyt** [reneweb/react-native-tensorflow#29]
When trying to integrate a pretrained tensorflow model with expo (react-native), the following error occurs within these lines:

```
   _graph = async () => {

        var preder2 = null;
        var items = "";
        this.setState({result: "", value: null});

        let result = await ImagePicker.launchImageLibraryAsync({
            allowsEditing: true,
            aspect: [4, 3],
            base64: true,
        });

        if (!result.cancelled) {
            this.setState({ image: result});
        };

        try {
            const tfImageRecognition = new TfImageRecognition({
                model: require('./assets/tensorflow_inception_graph.pb'),
                labels: require('./assets/tensorflow_labels.txt')
            });
    
            const results = await tfImageRecognition.recognize({
              image: this.state.image
            }); 
            results.forEach(
              result => ((preder2 = result.confidence), (items = result.name))
            );
            await tfImageRecognition.close();  
            this.setState({
              result: items,
              value: preder2 * 100 + "%"
            });
            console.log(this.state.result);
          } catch (err) {
            this.setState({
              result: "No Internet",
              value: "Please connect to the internet"
            });
            console.log(err);
          }
    }
```

Which generates the following error

> 10:10:40 AM: undefined is not an object (evaluating 'RNImageRecognition.initImageRecognizer')

I have been trying to find the reason why this is not working but I cannot find a definite solution. The relative paths linking to the assets are correct and the extensions are present in the app.json. Furthermore the model is trained using the tensorflow api which should make it compatible with the react-native implementation.

I observed that after running
```
const tfImageRecognition = new TfImageRecognition({

                model: require('./assets/tensorflow_inception_graph.pb'),
                labels: require('./assets/tensorflow_labels.txt')

            });
```
The code changed immediately to "catch(err)" branch, which means it could not load the model and labels?

I am using expo SDK version 28.0.0, Expo XDE and react-native-tensorflow version ^0.1.8

**Comment by NazerkeT** [comment]
I am struggling with the same problem as well. Is there somebody to help?

**Comment by SpQuyt** [comment]
Well I stopped using expo and then I used react-native init, then react-native run-android.
And it worked.
I don't know why but the most suitable answer for this is that this library cannot be used in expo?

**Comment by Isti01** [comment]
I ran into the same problem with the react-native init then react-native run-android method.
The relative path links are correct too.
The package versions that i am using:

`

    "react": "16.3.1",
    "react-native": "0.55.4",
    "react-native-tensorflow": "0.1.8"

`

**Comment by Isti01** [comment]
I accidentally forgot to link the package.
`$ react-native link react-native-tensorflow`
maybe you too.

**Comment by reneweb** [comment]
sorry for late reply - @SpQuyt it is indeed not possible to use native modules with expo, thus this lib can't be used with it

**Comment by SpQuyt** [comment]
@reneweb
